rdm-tools.php

```diff
@@ -1337,10 +1320,26 @@ function loadData() {
           }
         });
       }
-
+	  if (result.pokestops != null) {
+        result.pokestops.forEach(function(item) {
+          pokestops.push(item);
+          if (settings.showPokestopsRange === true) {
+            var marker = L.circle([item.lat, item.lng], {
+              color: 'green',
+              radius: 70,
+              opacity: 0.2
+            }).addTo(map);
+            marker.tags = {};
+            marker.tags.id = item.id;
+            marker.bindPopup("<span>ID: " + item.id + "</span>").addTo(pokestopRangeLayer);
+          }
+        });
+      }
       if (result.spawnpoints != null) {
         result.spawnpoints.forEach(function(item) {
           spawnpoints.push(item);
+          if (item.despawn_sec != null){
```

Comment:
Instead of duplicating the marker generation here with a switch for despawn_sec null, we should do something like if (item.despawn_sec != null) { var color = blue } else { var color = red }

Then pass the var to the marker builder. currently youre checking twice for each point, and with spawns being the most prevalant data i'd like to keep it as efficient as possible